docs/api/buckets.rst

```diff
@@ -300,3 +300,99 @@ Retrieving all buckets
                 }
             ]
         }
+
+
+.. buckets-default-id:
+
+Personal bucket «default»
```

Comment:
We should add a note about implicit creation of both the default bucket and the collection in it, when using default and not when using the bucket id.